src/components/photoViewer/PhotoViewer.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```

Comment:
Please:

1. Use double quotes (`"`) rather than single quotes (`'`) for strings
2. Put a semicolon (`;`) at the end of the line